src/nomad_simulations/schema_packages/physical_property.py

```diff
@@ -107,13 +55,8 @@ class PhysicalProperty(ArchiveSection):
         Label for additional classification of the physical property. Example: an `ElectronicBandGap`
         can be labeled as `'DFT'` or `'GW'` depending on the methodology used to calculate it.
         """,
-        # ! add more examples in the description to improve the understanding of this quantity
-    )
+    )  # TODO: specify use better
```

Comment:
agree, maybe we can wait and see some natural desired usages of this in our examples and then decide how to clarify

Reply:
Looking at our schema at large, we mostly use labels to define quick catch alls.
That is to say, either you scan several pieces of metadata, or you unite them into a label.
The example above reads more so as `type`. What do you think?

Reply:
```suggestion
   Short-hand for quickly distinguishing kinds of properties.
   Is typically mapped from some collection of metadata descriptors.
```

Reply:
I think both type and label are useful, and I am not sure of the precise usage. For me it would be easier to leave a TODO until we implement more examples and then update the description later based on our perspective now. But if you already have an idea, then change the description and also leave the/a TODO

Reply:
Well, this was my suggestion. There are already some examples in the new schema, but I'm fine building up more.